website/docs/docs/cloud/manage-access/invite-users.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Invite users to dbt Cloud"
+description: "Learn how to manually invite users to dbt Cloud"
+id: "invite-users"
+sidebar: "Invite users"
+---
+
+dbt Cloud makes it easy to invite new users to your environment out of the box. This feature is available to all dbt Cloud customers on Teams or Enteprise plans (Developer plans are limited to a single user).
+
+## Invite new users
+
+You must have proper permissions to invite new users:
+
+- [**Teams accounts**](/docs/cloud/manage-access/self-service-permissions) &mdash; must have `member` or `owner` permissions.
+- [**Enterprise accounts**](/docs/cloud/manage-access/enterprise-permissions) &mdash; must have `admin`, `account admin`, `project creator`, or `security admin` permissions.
+
+To invite users: 
+
+1. Open the **Account Settings** from the gear menu in your dbt Cloud account.
+2. From the left sidebar, select **Users**.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/users-screen.png" title="The user screen in settings"/>
+
+3. Click on **Invite Users**.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/invite-users.png" title="The invite users pane"/>
+
+4. In the **Email Addresses** field, enter the email addresses of the users you would like to invite separated by comma, semicolon, or a new line.
+5. Select the license type for the batch of users from the **License** dropdown.
+6. Select the group(s) you would like the invitees to belong to.
+7. Click **Send Invitations**.
+    - If the list of invitees exceeds the number of licenses your account has availabe, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 
+
+
+## User experience
+
+dbt generates and sends the email to the addresses defined in dbt Cloud. Be sure to allow traffic from `support@getdbt.com` to prevent emails from going to spam or being blocked (this is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses)).
```

Comment:
is it 'dbt' or 'dbt Cloud generates and sends..
```suggestion
dbt Cloud generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses).
--
dbt generates and sends the email to the addresses defined in dbt Cloud. Be sure to allow traffic from `support@getdbt.com` to prevent emails from going to spam or being blocked (this is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses)).
```

Reply:
Do we want to be this general? Do all emails the platform generates come from the support email address? If not, I think we should specify here that this is only for this workflow.

Reply:
hm not sure tbh (e.g. verify emails for cloud). but good flag to specify. maybe:

>dbt Cloud generates and sends invitation emails from `support@getdbt.com` to the specified addresses. Make sure traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses).